packages/real-hoist/package.json

```diff
@@ -38,6 +38,8 @@
   "devDependencies": {
     "@pnpm/lockfile-file": "workspace:4.2.6",
     "@pnpm/logger": "^4.0.0",
-    "@pnpm/real-hoist": "workspace:0.1.6"
+    "@pnpm/real-hoist": "workspace:0.1.6",
+    "vite": "^2.8.4",
```

Comment:
Should be no need to install `vite`. `vite` is now a dependency of `vitest`

Reply:
Sure, let me remove!